lib/services/local/resource.go

```diff
@@ -101,6 +101,10 @@ func itemsFromResource(resource types.Resource) ([]backend.Item, error) {
 		item, err = itemFromSAMLConnector(r)
 	case types.ProvisionToken:
 		item, err = itemFromProvisionToken(r)
+	case *types.DeviceV1:
+		// devices should be ignored here and instead restored
+		// using enterprise resource bootstrapper.
+		return nil, nil
```

Comment:
Noted. Thanks for the context. Fixed in https://github.com/gravitational/teleport/pull/24963/commits/79c0796833d4f344d1633935557589111b9535a1 by returning empty slice: `return []backend.Item{}, nil`.

However, this has to be included in this PR otherwise if this PR is merge and until "bootstrap" is supported `tctl get all > backup.yml && teleport start --bootstrap=backup.yml` will not not work as we return `NotImplemented` error below this block.